ttnn/cpp/ttnn/operations/data_movement/common/common.cpp

```diff
@@ -4,10 +4,27 @@
 
 #include "ttnn/cpp/ttnn/operations/data_movement/common/common.hpp"
 #include "ttnn/cpp/ttnn/operations/data_movement/pad/pad.hpp"
+#include "ttnn/cpp/ttnn/operations/data_movement/squeeze/squeeze.hpp"
 
 namespace ttnn {
 namespace operations {
 namespace data_movement {
+    ttnn::Tensor squeeze_to_le_4D(const ttnn::Tensor& tensor) {
```

Comment:
le_4d? Name is a bit weird no?

Reply:
`le` = less than or equal, informed by latex syntax 😛 we could change it to be more verbose if you think it will be confusing to people who read it down the line.

Reply:
that's fine